internal/experimental/registry/client.go

```diff
@@ -252,7 +253,7 @@ func (c *Client) Pull(ref string, options ...PullOption) (*PullResult, error) {
 		switch d.MediaType {
 		case ConfigMediaType:
 			configDescriptor = &d
-		case ChartLayerMediaType:
+		case ChartLayerMediaType, LegacyChartContentLayerMediaType:
```

Comment:
It might be better to put this into its own `case`, so that we can add some log warning that the chart is using a legacy type:
```
fmt.Fprintf(c.out, "Warning: chart media type %s is deprecated\n", LegacyChartLayerMediaType)

Reply:
ack